### PR TITLE
Operator formatting part 3

### PR DESCRIPTION
**Pull request by thomasfanell**
Formatted operators in
* source/geometry

### CODE REVIEW

**Comment by marcusnaslund** [comment]
I guess you *could* put `<` and `>` operators above `==` and `!=`, then the latter would align up nicely with `>=` and `<=`. This is true for a lot of covers in this PR.

**Comment by thomasfanell** [comment]
ok i'm on it

File: source/geometry/Quaternion.ooc [hunk 1]
```diff
@@ -346,6 +332,4 @@ Quaternion: cover {
 	}
 	kean_math_quaternion_new: unmangled static func (w, x, y, z: Float) -> This { This new(w, x, y, z) }
 }
-operator * (value: Float, other: Quaternion) -> Quaternion {
-	other * value
-}
+operator * (value: Float, other: Quaternion) -> Quaternion { other * value }
```
Comment (marcusnaslund) [inline source/geometry/Quaternion.ooc]: How about an empty line after classes/covers?

Comment (thomasfanell) [inline source/geometry/Quaternion.ooc]: I considered that, but since it's nothing below the operators, I let them be. Should I insert a new line?

Comment (marcusnaslund) [inline source/geometry/Quaternion.ooc]: Nah I'm ok with this.

**Pull request merged**